### PR TITLE
Added CLI link in readme

### PR DESCRIPTION
**Pull request by nirajpandkar**

### CODE REVIEW

**Comment by nirajpandkar** [comment]
@hemanth I hope the position in the readme is satisfactory.

**Comment by hemanth** [comment]
Looks nice, we can do a thing, I shall add you as an npm owner to `git-tips` would you mind renaming `gtips` to `git-tips`?

**Comment by nirajpandkar** [comment]
So let me get this clear, you want me to - 
1. Rename gtip -> git-tips in the readme
2. Github-link -> npm link 

That's it right?

**Comment by nirajpandkar** [comment]
Or are you talking about adding me as a collaborator in git-tips npm package? And then I should make the corresponding changes and push it to the new package?

**Comment by nirajpandkar** [comment]
Ping?

**Comment by hemanth** [comment]
> adding me as a collaborator in git-tips npm package

Yes ^

**Comment by nirajpandkar** [comment]
Sure. I haven't done this before.. What all would I have to do?
1. Edit `package.json` to change the name.

**Comment by hemanth** [comment]
Get me your npm handle.

After I add you to `git-tips` npm, could can `unpublish` `gtips`.

I shall create a `git-tips` repo for node module in here and then you can push the code to it.

What say?

**Comment by hemanth** [comment]
Or You can just rename your module and publish it as `git-tips`, naming makes sense, it sounds better than `gtips`?

**Comment by nirajpandkar** [comment]
Yes I agree about the naming :+1: I'll just rename my module to git-tips and update the PR.

**Comment by nirajpandkar** [comment]
Phew :sweat_smile: After a lot of refactoring, I've managed to change the name to `git-tip`. Deprecated gtip and also changed the link to npm package :+1: 
**Note**: Since the CLI outputs one tip at a time without the arguments I thought it sensible to name it git-tip without the extra 's'.

Updated the PR.

**Comment by hemanth** [comment]
Heh heh nice!

Should we add a tools section and add this? 🤔

**Comment by nirajpandkar** [comment]
Can't view the emoticon you just posted! And yeah will add a 'Tools' sub-category :+1:

**Comment by nirajpandkar** [comment]
How does that look? Nice to see a python implementation. Hope they complete the tool :smile:

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 [English](http://git.io/git-tips) | [中文](https://github.com/521xueweihan/git-tips)
 
+### Tools
```
Comment (hemanth) [inline README.md]: Sorry, but` __Tools:__` thanks!

Comment (nirajpandkar) [inline README.md]: Done :+1:

**Comment by nirajpandkar** [comment]
Anything to make it look good :smile:

**Comment by nirajpandkar** [comment]
Ping? @hemanth

**Comment by hemanth** [comment]
tLWWnZB7CLhF6

**Pull request merged**